CONTRIBUTING.md

```diff
@@ -74,6 +85,46 @@ branches that minor and patch releases are tagged. In some cases, it may
 be necessary to open PRs for bugfixes directly against stable branches, but
 this should generally not be the case.
 
+### Support and guarantees
+
+Cluster API Provider Metal3 maintains the most recent release/releases for all supported API and contract versions. Support for this section refers to the ability to backport and release patch versions;
+[backport policy](#backporting) is defined above.
+
+- The API version is determined from the GroupVersion defined in the top-level `api/` package.
+- The EOL date of each API Version is determined from the last release available once a new API version is published.
+
+| API Version  | Supported Until                                                              |
+| ------------ | ---------------------------------------------------------------------------- |
+| **v1beta1**  | TBD (current stable)                                                         |
+| **v1alpha5** | EOL since 2022-09-30 ([apiVersion removal](#removal-of-v1alpha5-apiversion)) |
+
+- For the current stable API version (v1beta1) we support the two most recent minor releases; older minor releases are immediately unsupported when a new major/minor release is available.
+- For older API versions we only support the most recent minor release until the API version reaches EOL.
+- We will maintain test coverage for all supported minor releases and for one additional release for the current stable API version in case we have to do an emergency patch release.
+  For example, if v1.2 and v1.3 are currently supported, we will also maintain test coverage for v1.1 for one additional release cycle. When v1.4 is released, tests for v1.1 will be removed.
```

Comment:
It would make more sense to give examples in upcoming versions than past ones.

This sentence also means were going to support 1.2, 1.3 and 1.4 on top of main as well. I thought we agreed we do two releases + main?